components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/dao/ApiMgtDAO.java

```diff
@@ -703,6 +704,120 @@ public void addSubscriber(Subscriber subscriber, String groupingId) throws APIMa
         }
     }
 
+    /**
+     * Derives info about monetization usage publish job
+     *
+     * @return ifno about the monetization usage publish job
+     * @throws APIManagementException
+     */
+    public MonetizationUsagePublishInfo getMonetizationUsagePublishInfo() throws APIManagementException {
+
+        Connection conn = null;
+        ResultSet rs = null;
+        PreparedStatement ps = null;
+        try {
+            conn = APIMgtDBUtil.getConnection();
+
```

Comment:
remove extra space

Reply:
Fixed in 0cb422c3d49f3c8e3e3d4e02e156665fd004836b